utils/displayAsciiBanner.ts

```diff
@@ -13,8 +13,8 @@ import type { Config } from '#types/config-types.js';
  * @returns {void}
  */
 const displayAsciiBanner = (config: Config): void => {
-    figlet(config.SERVICE_NAME || 'Service', (err: Error | null, data?: string | undefined) => {
-        if (err || !data) {
+    figlet(config.SERVICE_NAME ?? 'Service', (err: Error | null, data?: string  ) => {
+        if (err !== null || data === undefined ) {
```

Comment:
Strange, this change is more restrictive and will not catch null, empty string, or other falsy values. original code covers all falsy values: 
```
data === undefined
data === null
data === '' (empty string)
data === false
data === 0
and any other falsy value
```

Our intention is to show an error whenever data is missing or unusable (falsy), we should stick with the original.